### ISSUE TITLE
Accuracy results are worse when training with tensorflow>1.6

### ISSUE THREAD

**Issue by alexlee-gk** [alexlee-gk/video_prediction#13]
The accuracy results from the pretrained models were obtained with tensorflow 1.6. These accuracy results can't be obtained when training with tensorflow version 1.12. This only affects training. The pretrained models still generate correct predictions with tensorflow 1.12.

**Comment by alexlee-gk** [comment]
The issue seems to be related to the cudnn version and not the tensorflow version. The final training loss is worse when using cudnn versions 7.3.1.20 or 7.4.1.5, compared to when using versions 7.3.0.29 and below. For the best speed and experimental results, we recommend using cudnn version 7.3.0.29 and any tensorflow version >= 1.9 and <= 1.12.
